### ISSUE TITLE
Is this spec ready to move into a main W3C working group?

### ISSUE THREAD

**Issue by theres-waldo** [bokand/ViewportAPI#74]
I was wondering if this spec is ready to graduate into a main W3C working group (such as the CSS WG).

If not, are there specific issues or tasks blocking such a move?

**Comment by frivoal** [comment]
I think it's actually past time it moved to a WG (and the CSSWG is a good one). Regardless of whether there are bugs left, this has traction and shipping implementations. It should get the broader exposure of a WG, and that IPR protection that comes along with it.

If anyone thinks this API is harmful, that could change things, but I'm not aware of any such position.

**Comment by bokand** [comment]
Agreed. Any advice on how to start that process? I can take a look at turning this into some PRs against the appropriate spec (I guess CSSOM View?) but wondering if there's any other steps that need to be taken first?

**Comment by frivoal** [comment]
I'd suggest filing an issue in the CSSWG's repo suggesting adoption of the spec, with a pointer to the spec (and it's explainer if it has one), and adding the "Agenda+" label to get it some attention.

**Comment by bokand** [comment]
Thanks @frivoal, I've filed https://github.com/w3c/csswg-drafts/issues/6339 but aren't a contributor so can't add the label.

**Comment by bokand** [comment]
Just to update - I'm looking to fix the remaining issues with the draft spec in #66 and #65 - once I have those merged I'll put up a PR to merge this into CSSOM-View. See #80

**Comment by bokand** [comment]
Closing per #90
